src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -177,13 +177,12 @@ private Void deleteResourceInner(AuthenticatedUserRequest userReq,
     }
 
     @Override
-    public Map<IamRole, String> createDatasetResource(AuthenticatedUserRequest userReq, UUID datasetId)
+    public void createDatasetResource(AuthenticatedUserRequest userReq, UUID datasetId)
```

Comment:
Would it make sense to return the resource object that was created?

Reply:
I do not see a need to return anything. The next step - "CreateDatasetAuthzIamPolicyStep" returns the policy map, which is what is used in the following steps.